Analysis/Tasks/PWGHF/HFTrackIndexSkimsCreator.cxx

```diff
@@ -32,8 +33,57 @@ using namespace o2::aod;
 using namespace o2::analysis;
 using namespace o2::analysis::hf_cuts_single_track;
 
+/// Event selection
+struct HfProduceSelCollisions {
+
+  Produces<aod::HFSelCollision> rowSelectedCollision;
+
+  Configurable<bool> b_dovalplots{"b_dovalplots", true, "fill histograms"};
+  Configurable<std::string> s_trigger_class{"trigger_class", "kINT7", "trigger class"};
+  int trigger_class = std::distance(aliasLabels, std::find(aliasLabels, aliasLabels + kNaliases, s_trigger_class.value.data()));
+
+  HistogramRegistry registry{
+    "registry",
+    {{"h_events", "Events;;entries", {HistType::kTH1F, {{3, 0.5, 3.5}}}}}};
```

Comment:
Fix the variable names. (camelCase, no underscores, no type prefixes, `h` in histogram names are fine)

Reply:
done in https://github.com/AliceO2Group/AliceO2/pull/6016/commits/6f14765ffdb8058b18276786f72da58a92d36d48. I used snake_case because it was used in the `HfProduceSelTrack` class, so maybe we could then modify this also there in a next iteration